### PR TITLE
all: JacocoMerge must run after grpc-interop-testing's tests

### PR DESCRIPTION
**Pull request by ejona86**
Otherwise the executionData would be out-of-date.

### CODE REVIEW

File: all/build.gradle [hunk 1]
```diff
@@ -50,6 +50,7 @@ javadoc {
 
 task jacocoMerge(type: JacocoMerge) {
     dependsOn(subprojects.jacocoTestReport.dependsOn)
+    dependsOn(project(':grpc-interop-testing').jacocoTestReport.dependsOn)
     mustRunAfter(subprojects.jacocoTestReport.mustRunAfter)
```
Comment (dapengzhang0) [inline all/build.gradle:54]: Not sure if need `mustRunAfter(project(':grpc-interop-testing').jacocoTestReport.mustRunAfter)` as well.

Comment (voidzcy) [reply all/build.gradle:54]: `dependsOn` seems to be more reasonable than `mustRunAfter`. In terms of ordering for the two tasks, they are the same, you don't need to have both. If A dependsOn B, then A mustRunAfter B (plus other stuffs useful for incremental build/tasks).

**Review by dapengzhang0 — APPROVED** [review]

File: all/build.gradle (hunk 1 above)
Comment (ejona86) [reply all/build.gradle:54]: mustRunAfter is for things like clean+build. It is important for clean to run before build, but build does not have a dependency on clean.

Comment (ejona86) [reply all/build.gradle:54]: Oh, you're right. I see what you're saying now. I do need to copy those through. Things like `test` may be a mustRunAfter for merge.

**Review by dapengzhang0 — APPROVED** [review]

**Pull request merged**